creativecloud/scripts/utils.js

```diff
@@ -50,6 +49,19 @@ const miloLibs = setLibs('/libs');
 const { createTag, localizeLink, getConfig, loadStyle, createIntersectionObserver } = await import(`${miloLibs}/utils/utils.js`);
 export { createTag, loadStyle, localizeLink, createIntersectionObserver, getConfig };
 
+function defineDeviceByScreenSize() {
```

Comment:
We already have this function defineDeviceByScreenSize in decorate.js of CC. We can import it from there in place of duplicating the lines of code